server/config/game/specialists.json

```diff
@@ -624,22 +626,22 @@
                     "addNaturalResourcesOnTick": -1
                 },
                 "local": {
-                    "manufacturing": 5,
-                    "weapons": 5
+                    "manufacturing": 10,
+                    "weapons": 4
                 }
             }
         },
         {
             "id": 8,
             "key": "power-generator",
-            "name": "Scanner Scrambler",
-            "description": "Prevents other players from seeing how many ships are garrisoned at the star.",
+            "name": "Scanning Jammer",
+            "description": "A cutting-edge enigma device that shrouds the star system in a veil of cosmic mystery, rendering the count of star-cruisers harbored within imperceptible to adversarial cosmic entities.",
```

Comment:
On more thought, there is definitely an argument to making the game more accessible and easily understood. That we've found with some play testing.
I understand wanting to add some thematical ambience. And ChatGPT has its place. But

"Enigma device...harbored within imperceptible to adversarial cosmic entities"

Is very dense, vague and confusing.
What does this do? Does it hide infrastructure? Specialists? From who? Oh, just people I'm at war with?

"hides infrastructure and ship count from everyone but you." Is easy to grasp and understand, especially for new players and non English natives. 

When someone is looking what something is. They _dont_ want to read a wall of text or a novel.

Reply:
I agree.

Here is the revised text:
A megastructure that transmits a scanner disruption frequency concealing the number of ships in a star system, preventing others from scoping out your fleet."